lib/Config.js

```diff
@@ -1320,6 +1321,18 @@ class Config extends EventEmitter {
         }
 
         this.lifecycleRoleName = config.lifecycleRoleName || null;
+        this.multiObjectDeleteConcurrency = constants.multiObjectDeleteConcurrency;
+        if (!isNaN(config.multiObjectDeleteConcurrency)) {
```

Comment:
what is the point of checking for `NaN` here: `Number.parseInt` will handle it and return an error/NaN when needed anyway?